crates/subspace-runtime/src/lib.rs

```diff
@@ -439,6 +443,167 @@ impl pallet_sudo::Config for Runtime {
+    type Consideration = HoldConsideration<
+        AccountId,
+        Balances,
+        PreImageHoldReason,
+        LinearStoragePrice<PreimageBaseDeposit, PreimageByteDeposit, Balance>,
+    >;
+    type Currency = Balances;
+    type ManagerOrigin = EnsureRoot<AccountId>;
+    type RuntimeEvent = RuntimeEvent;
+    type WeightInfo = pallet_preimage::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * SubspaceBlockWeights::get().max_block;
+    pub const MaxScheduledPerBlock: u32 = 50;
+    // Retry a scheduled item every 10 blocks (2 minutes) until the preimage exists.
+    pub const NoPreimagePostponement: Option<u32> = Some(10);
+}
+
+impl pallet_scheduler::Config for Runtime {
+    type MaxScheduledPerBlock = MaxScheduledPerBlock;
+    type MaximumWeight = MaximumSchedulerWeight;
+    type OriginPrivilegeCmp = EqualPrivilegeOnly;
+    type PalletsOrigin = OriginCaller;
+    type Preimages = Preimage;
+    type RuntimeCall = RuntimeCall;
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeOrigin = RuntimeOrigin;
+    type ScheduleOrigin = EnsureRoot<AccountId>;
+    type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
+}
+
+// TODO: update params for mainnnet
+pub type EnsureRootOr<O> = EitherOfDiverse<EnsureRoot<AccountId>, O>;
+pub type AllCouncil = EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>;
+pub type TwoThirdsCouncil = EnsureProportionAtLeast<AccountId, CouncilCollective, 2, 3>;
+pub type HalfCouncil = EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>;
+
+// TODO: update params for mainnnet
+parameter_types! {
+    // launch period for public referendum. Currently set to 2 days
+    pub LaunchPeriod: BlockNumber = 28_800;
+    // Voting period for democracy. Currently set to 1 day
+    pub VotingPeriod: BlockNumber = 14_400;
+    // Voting period for fast track. Currently set to 2 hours.
+    pub FastTrackVotingPeriod: BlockNumber = 1_200;
+    // Allow instant execution of proposals.
+    pub const InstantAllowed: bool = true;
+    pub const MinimumDeposit: Balance = 1000 * SSC;
+    // Enactment period for referendum. Currently set to 1 day.
+    pub EnactmentPeriod: BlockNumber = 14_400;
+    // Cool off period for council if the previous proposal was vetoed.
+    // Currently set to 1 day
+    pub CooloffPeriod: BlockNumber = 14_400;
+    pub const MaxProposals: u32 = 100;
+    pub const MaxVotes: u32 = 100;
+}
+
+type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
+
+pub struct DemocracySlash;
+impl OnUnbalanced<NegativeImbalance> for DemocracySlash {
+    fn on_nonzero_unbalanced(slashed: NegativeImbalance) {
+        Balances::resolve_creating(&TreasuryAccount::get(), slashed);
+    }
+}
+
+impl pallet_democracy::Config for Runtime {
+    type BlacklistOrigin = EnsureRoot<AccountId>;
+    // To cancel a proposal before it has been passed, must be root.
```

Comment:
Optional: Slashing seems like an important thing to document:
https://docs.rs/pallet-democracy/latest/pallet_democracy/pallet/trait.Config.html#associatedtype.CancelProposalOrigin

```suggestion
    // To cancel a proposal before it has been passed and slash its backers, must be root.
```